### ISSUE TITLE
Problem about reproducing

### ISSUE THREAD

**Issue by hezekiah94** [CrossmodalGroup/NAAF#17]
The Flickr30k checkpoint you provided (rsum: 507.9) was trained with `batch_size=256`, not at all using the hyperparameter `batch_size=128` you declared.

**Comment by CrossmodalGroup** [comment]
Yes, as we verified in the paper, a larger batch-size is better. You can also set the batch-size to 256 to run the code.

**Comment by CrossmodalGroup** [comment]
The batch_size 128 corresponds to the reported results in the paper (see Implementation Details), which is lower than our reproduced results (rSum: 507.9 using batch_size 256). I suggest you carefully read the papers.

Not only I, but others can reproduce this code. I have patiently answered your questions. Just so you know, I am doing this in good faith. But your attitude leaves a very negative impression, and I won't answer any more of your questions.
